<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="id_Label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DFKai-SB</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_Label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DFKai-SB</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>PASSWORD: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Application_Header">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Maiandra GD</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>FTP Application </string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>480</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Demi</family>
      <pointsize>16</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>10</hour>
      <minute>24</minute>
      <second>30</second>
      <year>2016</year>
      <month>9</month>
      <day>17</day>
     </datetime>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::MonthSection</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::UTC</enum>
    </property>
   </widget>
   <widget class="QLabel" name="host_Label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DFKai-SB</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HOST/IP:</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>530</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>9</hour>
      <minute>41</minute>
      <second>20</second>
      <year>2016</year>
      <month>9</month>
      <day>15</day>
     </datetime>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_PushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>471</width>
      <height>23</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="port_Label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DFKai-SB</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>PORT:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textInput_Host">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textInput_ID">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textInput_Password">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textInput_Port">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1135</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
